azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -283,6 +289,28 @@ internal class CallCompositeActivity : AppCompatActivity() {
         }
     }
 
+    private fun actionBarAccessibilitySetup() {
+        try {
+            findViewById<View>(R.id.action_bar)?.let { actionBarView ->
```

Comment:
Why <View> shouldn't be <ActionBarView>

Reply:
Its type is ToolBar, do we need to specifically indicate type when we just need to get only child?

Reply:
val viewGroup = actionBarView as ViewGroup
If you want ViewGroup, then pass ViewGroup into findViewById